server/test/controllers/admin/AdminImportControllerTest.java

```diff
@@ -420,6 +420,7 @@ public void saveProgram_preservesUniversalSettingOnImportedQuestions() {
           + "    \"acls\" : {\n"
           + "      \"tiProgramViewAcls\" : [ ]\n"
           + "    },\n"
+          + "    \"categories\" : [ ], \n"
```

Comment:
(note) You don't need to change anything here. Just an FYI in case you didn't know. Now that we are on Java 17 you can use the `"""` to do [multiline strings](https://www.baeldung.com/java-multiline-string).